.github/workflows/release_build_test.yaml

```diff
@@ -13,196 +13,78 @@ on:
         options:
           - Release
           - RelWithDebInfo
+      os:
+        type: choice
+        description: "Target OS on which the release tests will be run."
+        default: 'debian-11'
```

Comment:
12?

Reply:
We are currently experiencing problems with running all tests on `debian-12`. I've made a PR (#1834) to fix these issues and upgrade from `debian-11` to `debian-12` as our default OS across all workflows where we run tests (`diff` and `release build tests`).

Reply:
I would like to upgrade this after the `2.15.1` release and before the `2.16.0` release, what do you think?

Reply:
Ok